### PR TITLE
Add photo

### PR DESCRIPTION
**Pull request by ghost**
I add the photo

### CODE REVIEW

**Comment by prwatch** [comment]
DEMO: http://nolik2000.github.io/gci15.fossasia.org

File: _data/students.yml [hunk 1]
```diff
@@ -332,3 +332,11 @@
   blog: http://poprosturonin.com
   twitter: poprosturonin
   bio: Gamer, developer, designer from Poland.
+  
+  
+  - name: Nastya Kostiv
```
Comment (jigyasa-grover) [inline _data/students.yml]: Kindly see the format used. Remove the space before `- name`.

**Comment by jigyasa-grover** [comment]
@nolik2000 

Build failing.

Also squash commits.
[This](https://github.com/fossasia/gci15.fossasia.org/blob/gh-pages/GIT.md) might help.

**Comment by ghost** [comment]
it doesn't work

**Comment by ghost** [comment]
i just tried

**Comment by ghost** [comment]
help me please

**Comment by mananwason** [comment]
@nolik2000 what doesn't work?

**Comment by ghost** [comment]
https://help.github.com/articles/troubleshooting-github-pages-build-failures.

**Comment by jigyasa-grover** [comment]
@nolik2000 Just re-format according to the comment on your additions and it shall work, Don't Worry :)

**Comment by mananwason** [comment]
@nolik2000 Don't worry about that. Please squash the commits into one first. You can refer [this](https://github.com/fossasia/gci15.fossasia.org/blob/gh-pages/GIT.md). Cheers!

**Comment by ghost** [comment]
thank's I will try that

**Comment by mananwason** [comment]
:+1:

**Comment by jigyasa-grover** [comment]
@nolik2000 The error still persists.

**Comment by jigyasa-grover** [comment]
Closing this PR as a new one #318 has been filed by @nolik2000

**Pull request closed without merging**